src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts

```diff
@@ -1,18 +1,24 @@
+import { PaypalButtonStyleOptions } from './paypal-commerce-sdk';
+
 export interface PaypalCommercePaymentInitializeOptions {
-    overlay?: {
-        helpText?: string;
-        continueText?: string;
-    };
+    container: string;
+    style?: PaypalButtonStyleOptions;
+    submitForm(): void;
+    hidePaymentButton?(): void;
```

Comment:
Can we name this parameter as `onRenderButton` as well?

Reply:
Sure